benchbuild/source/variants/base.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict, Iterable
+
+import attr
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'attr'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)